File: ОАИП/LR_1-2/5.2.pas
var a, b, c, max, min, sub: integer;
begin
writeln('Введите массы гантелей качков');
read(a, b, c);
if (a>=b) and (a>=c) then
  max:=a;
if (b>=a) and (b>=c) then
  max:=b;
if (c>=a) and (c>=b) then
  max:=c;
if (a<=b) and (a<=c) then
  min:=a;
if (b<=a) and (b<=c) then
  min:=b;
if (c<=a) and (c<=b) then
  min:=c;
sub:=max-min;
write('Разница между гантелей самого сильного и самого слабого качков составляет ', sub, ' кг');
end.

File: ОАИП/LR_1-2/5.2.py
a = int(input('Введите массу гантели первого качка: '))
b = int(input('Введите массу гантели второго качка: '))
c = int(input('Введите массу гантели третьего качка: '))
if (a >= b) and (a >= c):
    max = a
elif (b >= a) and (b >= c):
    max = b
elif (c >= a) and (c >= b):
    max = c
if (a <= b) and (a <= c):
    min = a
elif (b <= a) and (b <= c):
    min = b
elif (c <= a) and (c <= b):
    min = c
sub = max - min
print(f'Разница между гантелей самого сильного и самого слабого качков составляет {sub} кг')
